spec/bundle/commands/install_command_spec.rb

```diff
@@ -16,15 +16,23 @@
   end
 
   context "when a Brewfile is found" do
+    let(:brewfile_contents) do
+      <<~HEREDOC
+        tap 'phinze/cask'
+        brew 'mysql', conflicts_with: ['mysql56']
+        cask 'google-chrome'
+        mas '1Password', id: 443987910
+        whalebrew 'whalebrew/wget'
+      HEREDOC
```

Comment:
```suggestion
      <<~EOS
        tap 'phinze/cask'
        brew 'mysql', conflicts_with: ['mysql56']
        cask 'google-chrome'
        mas '1Password', id: 443987910
        whalebrew 'whalebrew/wget'
      EOS
```

but generally: 👏 on making this a heredoc